schema.graphql

```diff
@@ -14,6 +14,7 @@ type SpotOrderChangeEvent {
 
 type SpotOrder {
   id: ID!
+  order_type: String
```

Comment:
What do you think of using `enum` for this? I think it'll be nicer in terms of usage and generated types.

You can see how they are used at https://github.com/enviodev/thunder-exchange-fuel/blob/2e2dfc025ccf38402640cf44aed2744b2db04855/schema.graphql#L1-L4